test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -51,6 +54,11 @@ def send_message(self, message: str):
         type(ChatComponents.MESSAGE_INPUT.value, message)
         press_enter(ChatComponents.MESSAGE_INPUT.value)
         
+    def select_emoji(self):
```

Comment:
Let's make the function name more representative
```suggestion
    def select_the_emoji_in_suggestion_list(self):
```